src/services/AbstractDFG.jl

```diff
@@ -1216,3 +1216,14 @@ function getSummaryGraph(dfg::G)::LightDFG{NoSolverParams, DFGVariableSummary, D
     # end
     return summaryDfg
 end
+
+
+
+## features for broadcasting
+import Base: length, iterate
+
+# to allow stuff like `getFactorType.(dfg, [:x1x2f1;:x10l3f2])`
+# not working properly yet -- trying to immitate `add=+; add.(3,[1;2])`
+# https://docs.julialang.org/en/v1/manual/interfaces/#
+length(::AbstractDFG) = 1
+iterate(dfg::AbstractDFG, state::Int=0) = (dfg,state+1)
```

Comment:
this would be needed for iterators and not for `getFactorType.(dfg, [:x1x2f1;:x10l3f2])`